### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by faisalakb**
Could you check the PR? I updated the following features on my Portfolio website :

- Page title – changing the page title name. 
   (https://github.com/faisalakb/Portfolio-website1/blob/99d10c3f7105f7e1c4300aa58939512601737312/index.html#L17 )

- Image text alternatives – updated all image text alternatives (alt attribute). 
( https://github.com/faisalakb/Portfolio-website1/blob/99d10c3f7105f7e1c4300aa58939512601737312/index.html#L40-L44 ).

- Text headings- no issues found.

- Color contrast- no issues found

- Resize- On the nav bar- increase the font size of the items for clear visibility.
( https://github.com/faisalakb/Portfolio-website1/blob/99d10c3f7105f7e1c4300aa58939512601737312/style.css#L1042 )

- Interaction- Add label tag in forms and set it as hidden for blinds to access the input field of the forms.
( https://github.com/faisalakb/Portfolio-website1/blob/99d10c3f7105f7e1c4300aa58939512601737312/index.html#L198-L200 )

- Moving content- no issues found.

- Multimedia- no issues found.

- The basic structure of the page- no issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 <body>
   <header class="header1 ">
```
Comment (gemmen29) [inline index.html:16]: - [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @faisalakb, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct color contrast ✔️ 
- Correct website's accessibility 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
Thanks for the time.

**Pull request merged**